modules/ROOT/pages/configure-host.adoc

```diff
@@ -31,12 +31,30 @@ For example, depending on your setup, select one of the following: AS2, DUNS num
 
 . Click *New* within the *Certificates* section to begin to upload any needed digital certificates (such as an X.509 certificate in PKCS12 file format for the AS2 transport protocol).
 
-. Click *Choose file* to select the certificate (typically a `.PEM` file), and click *Upload certificate* to upload the chosen digital certificate file. See xref:Certificates.adoc[Certificates] for more info on configuring certificates in Partner Manager.
+. Click *Choose file* to select the certificate (typically a `.PEM` file), and click *Upload certificate* to upload your chosen certificate file. See xref:Certificates.adoc[Certificates] for more info on configuring certificates in Partner Manager.
++
+NOTE: If a single certificate is uploaded, Partner Manager prevents you from deleting your only certificate.
++
+. Click *New* from within _Endpoints: Receive from partners_ or _Endpoints: Targets at <partner name>_ for the Host, and select your protocol type.
 
-. (Optional) Click *Edit* within the *Payload storage* section to change any of the information that was completed when the payload storage was set up. For example, URL location, timeout settings, and acknowledgments.
+. Set up your endpoint based on the protocol type you selected.
++
+For more information on your protocol type refer to:
+
+* xref:endpoint-sftp-receive-target.adoc[SFTP Receive Endpoint Settings]
+
+* xref:endpoint-https-send.adoc[HTTP and HTTPS Send Configuration]
++
+. Save when you are done.
+
+. If you need to change any of the information that was completed when the payload storage was set up, click *Edit* within the *Payload storage* section.
+
+For example, URL location, timeout settings, and acknowledgments.
```

Comment:
This is a fragment. Please incorporate it into the sentence above or make it complete in itself.

Reply:
Fixed.